ui/main.qml

```diff
@@ -192,14 +192,15 @@ StatusWindow {
 
     Component.onCompleted: {
         Style.changeTheme(localAppSettings.theme, systemPalette.isCurrentSystemThemeDark())
-        setX(Qt.application.screens[0].width / 2 - width / 2);
-        setY(Qt.application.screens[0].height / 2 - height / 2);
-
         if (!localAppSettings.appSizeInitialized) {
-            width = Screen.desktopAvailableWidth - 125
-            height = Screen.desktopAvailableHeight - 125
+            width = Math.min(Screen.desktopAvailableWidth - 125, 1400)
+            height = Math.min(Screen.desktopAvailableHeight - 125, 840)
```

Comment:
@simonricoo So it turned out simply going with available width/height -125 doesn't work well for people using super big/multiple screens.

I've now put a max limit to 1400x840 for the initial load in case no size was set before. Those values are somewhat arbitrary, so let me know if you prefer other numbers here.